README.md

```diff
@@ -4,10 +4,27 @@ stackage
 [![Build Status](https://travis-ci.org/fpco/stackage.svg?branch=master)](https://travis-ci.org/fpco/stackage)
 
 "Stable Hackage": creating a vetted set of packages from Hackage.
+This repository is for package authors and maintainers to get their packages into Stackage.
+If you simply want to use Stackage as an end user, please follow the instructions on [http://www.stackage.org/](http://www.stackage.org).
 
-__NOTE__ This repository is for package authors and maintainers to get their packages into
-Stackage. If you simply want to use Stackage as an end user, please follow the
-instructions on [http://www.stackage.org/](http://www.stackage.org).
+We strongly recommend using the Haskell tool stack for doing builds, which
+includes built-in Stackage support: [stack](https://github.com/commercialhaskell/stack) [![Build Status](https://travis-ci.org/commercialhaskell/stack.svg?branch=master)](https://travis-ci.org/commercialhaskell/stack).
+
+
+Add your package
+----------------
+
+We welcome all packages, provided:
+
+* The package author/maintainer agrees to the [maintainers agreement](https://github.com/fpco/stackage/blob/master/MAINTAINERS.md).
+* The package is buildable and testable from Hackage. We recommend the [multi-ghc-travis](https://github.com/hvr/multi-ghc-travis), which ensures the package is not accidentally incomplete.
```

Comment:
It's in the user guide at:

http://docs.haskellstack.org/en/stable/GUIDE.html#travis-with-caching

Putting up a blog post for publicity is a good idea, I thought I'd done so already but apparently not. In any event, linking to this location is best since it's kept up-to-date.